modules/workers/locals.tf

```diff
@@ -88,6 +92,12 @@ locals {
 
       # Use provided image_id for 'custom' type, or first match for all shape + OS criteria
       image_id = (pool.image_type == "custom" ? pool.image_id : element(tolist(setintersection([
+        pool.image_type == "oke" ? 
+          setintersection(
+            lookup(var.image_ids, "oke", null), 
+            lookup(var.image_ids, format("%v", substr(pool.kubernetes_version, 1, length(pool.kubernetes_version))), null)
```

Comment:
Would [trimprefix](https://developer.hashicorp.com/terraform/language/v1.2.x/functions/trimprefix)`(pool.kubernetes_version, "v")` work in place of the `substr(...)`, for consistent output when present/absent?

Reply:
Unneeded `format()`?

Reply:
Thank you! Fixed.